.mergify.yml

```diff
@@ -25,17 +31,26 @@ pull_request_rules:
   - name: L10N - Auto Merge
     conditions:
       - and:
-        - files~=(strings.xml|l10n.toml)
+        - files~=^(l10n.toml|app/src/main/res/values[A-Za-z-]*/strings\.xml)$
+        # /!\ The line above doesn't prevent random files to be changed alongside
+        # l10n ones. That's why the additional condition exists below. For more
+        # information: https://docs.mergify.com/conditions/#how-to-match-lists
+        - -files~=^(?!(l10n.toml|app/src/main/res/values[A-Za-z-]*/strings\.xml)).+$
```

Comment:
I tested both regexes on [1]. Mergify uses the Python flavor[2]. I checked we can't make some code changes alongside string updates in [3]: it works as expected.

[1] https://regex101.com/
[2] https://docs.mergify.com/configuration/#regular-expressions
[3] https://github.com/mozilla-releng/staging-fenix/pull/11/checks?check_run_id=7410460256